findme/Contact.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Contact.swift
+//  findme
+//
+//  Created by Nicolas Mercier on 15/03/2016.
+//  Copyright © 2016 Maxime Signoret. All rights reserved.
+//
+
+import UIKit
+import MapKit
+
+class Contact : NSObject, MKAnnotation {
```

Comment:
Not an appropriate class name. class User better. Or create a User master class and a contact class which implements User.